plugins/languages/texpresso.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+  helpers.vim-plugin.mkVimPlugin config {
+    name = "texpresso";
+    originalName = "texpresso.vim";
+    defaultPackage = pkgs.vimPlugins.texpresso-vim;
+
+    maintainers = with maintainers; [nickhu];
+
+    extraOptions = {
+      texpressoPackage = mkOption {
+        type = with types; nullOr package;
+        default = pkgs.texpresso;
+        example = null;
+        description = ''
+          The `texpresso` package to use.
+          Set to `null` to not install any package.
+        '';
+      };
+    };
+
+    extraConfig = cfg: {
+      extraPackages = optional (cfg.texpressoPackage != null) cfg.texpressoPackage;
```

```suggestion
      extraPackages = [cfg.texpressoPackage];
```

Since [this commit](https://github.com/nix-community/nixvim/commit/f2f97d844bb39559f3356e209b49c92900d860b8), you can just dump `null` values to `extraPackages` and they will get properly ignored.